<!DOCTYPE HTML>
<html lang="en">
<head>
	<meta name="google-site-verification" content="uu3sMcpLHyf5aLCAKHeZ6MIygXX00_BaCq3gdmTM_G4"/>
	<meta name="description" content="Calvin Chi's personal website"/>
	<title>Calvin Chi</title>
	<link rel="stylesheet" href = "assets/css/style.css">
	<link rel="stylesheet" href = "assets/css/ammap.css">
	<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
	<link rel="icon" href="assets/img/cal.jpg">
	<script type="text/javascript" src="assets/js/ammap.js"></script>
	<script type="text/javascript" src="assets/js/worldLow.js"></script>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
	<script src = "assets/js/script.js"></script>
	<script>
		var map;
		// svg path for target icon
		var targetSVG = "M9,0C4.029,0,0,4.029,0,9s4.029,9,9,9s9-4.029,9-9S13.971,0,9,0z M9,15.93 c-3.83,0-6.93-3.1-6.93-6.93S5.17,2.07,9,2.07s6.93,3.1,6.93,6.93S12.83,15.93,9,15.93 M12.5,9c0,1.933-1.567,3.5-3.5,3.5S5.5,10.933,5.5,9S7.067,5.5,9,5.5 S12.5,7.067,12.5,9z";
		AmCharts.ready(function() {
			map = new AmCharts.AmMap();
		    map.imagesSettings = {
		        rollOverColor: "#CC0000",
		        rollOverScale: 3,
                    selectedScale: 3,
		        selectedColor: "#CC0000"
		    };
			map.areasSettings = {
				outlineThickness:0.1
			};
			var dataProvider = {
            	mapVar: AmCharts.maps.worldLow,
            	images: [
            		{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Taichung", latitude:24.147736, longitude:120.673648},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Kaohsiung", latitude:22.627278, longitude:120.301435},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Taipei", latitude:25.0330, longitude:121.5654},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Okinawa", latitude:26.501301, longitude:127.945404},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Seoul", latitude:37.566535, longitude:126.977969},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Hong Kong", latitude:22.396428, longitude:114.109497},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"San Francisco", latitude:37.774929, longitude:-122.419416},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"San Diego", latitude:32.715738, longitude:-117.161084},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Los Angeles", latitude:34.052234, longitude:-118.243685},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Seattle", latitude:47.606209, longitude:-122.332071},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Vancouver", latitude:49.282729, longitude:-123.120738},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Pittsburgh", latitude:40.440625, longitude:-79.995886},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"New York", latitude:40.712775, longitude:-74.005973},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Orlando", latitude:28.538335, longitude:-81.379236},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Cleveland", latitude:41.499320, longitude:-81.694361},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Columbus", latitude:39.961176, longitude:-82.998794},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Iowa City", latitude:41.661128, longitude:-91.530168},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Osaka", latitude:34.6937, longitude:135.5022},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Kyoto", latitude:35.0116, longitude:135.7680},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Page", latitude:36.9147, longitude:-111.4558},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Grand Canyon Village", latitude:36.0544, longitude:-112.1401},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Phoenix", latitude:33.4484, longitude:-112.0740},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Portland", latitude:45.5051, longitude:-122.6750},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Houston", latitude:29.7604, longitude:-95.3584},
          			{svgPath:targetSVG, zoomLevel:5, scale:0.5, title:"Las Vegas", latitude:36.1146, longitude:-115.1728}
          			]
            };
	    	map.dataProvider = dataProvider;
	    	map.showImagesInList = true;
	    	map.write("mapdiv");
		});
    </script>
</head>
<body>
	<section id = "about-section" class = "white-section">
		<div id = "cover"></div>
		<div id="about">
			<div id = "profile">
				<div id = "profile-photo">
					<img src="assets/img/profile.jpg" height="250px" style="border-radius:50%; margin:auto; display: block;">
				</div>
				<div id = "profile-caption">
					<h1>Calvin Chi</h1>
					<h3><i>calvin.chi at berkeley dot edu</i></h3>
					<br>
					<br>
					<div class = "profile-button-panel">
						<a class = "profile-button" href = "https://scholar.google.com/citations?hl=en&user=GvAm5k8AAAAJ" target = "_blank">Google Scholar</a>
						<a class = "profile-button" href = "https://github.com/CalvinTChi" target = "_blank">Github</a>
						<a class = "profile-button" href = "https://calvintchi.github.io/pdf/resume.pdf" target = "_blank">Resume</a>
					</div>
				</div>
			</div>
			<div id = "about-text">
				<p> I am an applied scientist at Amazon Advertising, working on semi-supervised and multitask learning for machine learning systems.

				<br>

				<br> I completed my Ph.D. in Computational Biology at UC Berkeley in 2020, advised by Haiyan Huang and Lisa Barcellos, and supported by the NSF Graduate Research Fellowship. My graduate work was at the intersection of applied machine learning and epidemiology/pharmacogenomics, which is summarized by this <a target="_blank" style="color:#3399ff" href="http://calvintchi.github.io/pdf/dissertation.pdf">dissertation</a>.

				<br>

				<br>My academic interests include the foundational topics of optimization, statistics, and machine learning; application areas such as vision, language, and biomedicine; and frameworks such as reinforcement learning and metalearning.</p>

				<p style="width:100%"><br><i>"Nature cannot be fooled."</i> - Richard Feynman</p>
			</div>
		</div>
	</section>
	<section id = "biography-section" class = "gray-section">
		<div class="navbar">
			<a class="nav-item" href="#about-section">About</a>
			<a class="nav-item" href="#biography-section">Biography</a>
			<a class="nav-item" href="#education-section">Education</a>
			<a class="nav-item" href="#industry-section">Industry</a>
			<a class="nav-item" href="#research-section">Publications</a>
			<a class="nav-item" href="#projects-section">Projects</a>
			<a class="nav-item" href="#resources-section">Resources</a>
		</div>
		<div id="biography">
			<div class="section-title">Biography</div>
			<div id="biography-text">
				<p> I grew up in Taichung, Taiwan as a third culture kid, and always had an interest in STEM. Due to my high school's emphasis on the liberal arts, I did not know about engineering as a career until my senior year. Since I thought medicine to be the most practical application of STEM, I committed my college career towards earning medical school admission. It was not until the interviews when I deeply realized medicine to be the wrong path, leading me to make my most risky career move by turning down my admissions offer. I later pursued a Ph.D. in Computational Biology to connect with statistics, computer science, and math, and finally realigned my direction with my passions. If you happen to find my <a target="_blank" style="color:#3399ff" href="http://www.ratemyprofessors.com/ShowRatings.jsp?tid=1749628">rate my professors profile</a>, it is a joke started by college classmates back in organic chemistry. The map indicates cities I have been to.</p>
			</div>
			<div id="mapdiv"></div>
		</div>
	</section>
	<section id = "education-section" class = "white-section">
		<div class = "section-title">Education</div>
		<div class="organization">
			<div class="organization-photo">
				<img src="assets/img/CWRU.jpg" height="250px" style="border-radius:50%; display: block; margin: auto;">
			</div>
			 <div class="organization-caption">  
				<br>
				<h2>Case Western Reserve University</h2>
				BS, Biochemistry, <i>summa cum laude</i>
				<br>
				2011 - 2015
			</div>
		</div>
		<div class="organization">
			<div class="organization-photo">
				<img src="assets/img/UCB.jpg" height="250px" style="border-radius:50%; display: block; margin: auto;">
			</div>
			<div class="organization-caption"> 
				<br>
				<h2>University of California, Berkeley</h2>
				PhD, Computational Biology
				<br>
				2015 - 2020
			</div> 
		</div>
	</section>
	<section id = "industry-section" class = "gray-section">
		<div class = "section-title">Industry</div>
		<div class="organization">
			<div class="organization-photo">
				<img src="assets/img/amazon.jpg" height="250px" style="border-radius:50%; display: block; margin: auto;">
			</div>
			 <div class="organization-caption"> 
			 	<br> 
				<h2>Amazon</h2>
				Applied Scientist Intern, summer 2019
				<br>
				Applied Scientist, 2020 - present
			</div>
		</div>
	</section>
	<section id = "research-section" class = "white-section">
		<div class = "section-title">Publications</div>
		<div class = "work">
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/research3.jpg" height="240px">
				</div>
				<div class = "work-text">
					<h2>Hypomethylation mediates genetic association with the major histocompatibility complex genes in Sj√∂gren's syndrome</h2>
					<p><b>Calvin Chi</b>, Kimberly E. Taylor, Hong Quach, Diana Quach, Lindsey A. Criswell, Lisa F. Barcellos</p>
					<p>PLoS ONE 16(4) 2021</p>
					<br>
					<br>
					<div class = "button-panel">
						<div type = "button" class = "button-abstract">Abstract
							<span class = "abstract-bubble">
								Differential methylation of immune genes has been a consistent theme observed in Sj√∂gren‚Äôs syndrome (SS) in CD4+ T cells, CD19+ B cells, whole blood, and labial salivary glands (LSGs). Multiple studies have found associations supporting genetic control of DNA methylation in SS, which in the absence of reverse causation, has positive implications for the potential of epigenetic therapy. However, a formal study of the causal relationship between genetic variation, DNA methylation, and disease status is lacking. We performed a causal mediation analysis of DNA methylation as a mediator of nearby genetic association with SS using LSGs and genotype data collected from 131 female members of the Sj√∂gren‚Äôs International Collaborative Clinical Alliance registry, comprising of 64 SS cases and 67 non-cases. Bumphunter was used to first identify differentially-methylated regions (DMRs), then the causal inference test (CIT) was applied to identify DMRs mediating the association of nearby methylation quantitative trait loci (MeQTL) with SS. Bumphunter discovered 215 DMRs, with the majority located in the major histocompatibility complex (MHC) on chromosome 6p21.3. Consistent with previous findings, regions hypomethylated in SS cases were enriched for gene sets associated with immune processes. Using the CIT, we observed a total of 19 DMR-MeQTL pairs that exhibited strong evidence for a causal mediation relationship. Close to half of these DMRs reside in the MHC and their corresponding meQTLs are in the region spanning the HLA-DQA1, HLA-DQB1, and HLA-DQA2 loci. The risk of SS conferred by these corresponding MeQTLs in the MHC was further substantiated by previous genome-wide association study results, with modest evidence for independent effects. By validating the presence of causal mediation, our findings suggest both genetic and epigenetic factors contribute to disease susceptibility, and inform the development of targeted epigenetic modification as a therapeutic approach for SS.
								<div class = "middle-dark-tooltip"></div>
								<div class = "middle-light-tooltip"></div>
							</span>
						</div>
						<a class="button" href = "https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248429" target = "_blank">Paper</a>
					</div>
				</div>
			</div>
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/research2.jpg" height="240px">
				</div>
				<div class = "work-text">
					<h2>Bipartite graph-based approach for clustering of cell lines by gene expression-drug response associations</h2>
					<p><b>Calvin Chi</b>, Yuting Ye, Bin Chen, Haiyan Huang</p>
					<p>Bioinformatics 2021</p>
					<br>
					<br>
					<div class = "button-panel">
						<div type = "button" class = "button-abstract">Abstract
							<span class = "abstract-bubble">
								In pharmacogenomic studies, the biological context of cell lines influences the predictive ability of drug-response models and the discovery of biomarkers. Thus, similar cell lines are often studied together based on prior knowledge of biological annotations. However, this selection approach is not scalable with the number of annotations, and the relationship between gene-drug association patterns and biological context may not be obvious. We present a procedure to compare cell lines based on their gene-drug association patterns. Starting with a grouping of cell lines from biological annotation, we model gene-drug association patterns for each group as a bipartite graph between genes and drugs. This is accomplished by applying sparse canonical correlation analysis (SCCA) to extract the gene-drug associations, and using the canonical vectors to construct the edge weights. Then, we introduce a nuclear norm-based dissimilarity measure to compare the bipartite graphs. Accompanying our procedure is a permutation test to evaluate the significance of similarity of cell line groups in terms of gene-drug associations. In the pharmacogenomics datasets CTRP2, GDSC2, and CCLE, hierarchical clustering of carcinoma groups based on this dissimilarity measure uniquely reveals clustering patterns driven by carcinoma subtype rather than primary site. Next, we show that the top associated drugs or genes from SCCA can be used to characterize the clustering patterns of haematopoietic and lymphoid malignancies. Finally, we confirm by simulation that when drug responses are linearly-dependent on expression, our approach is the only one that can effectively infer the true hierarchy compared to existing approaches.
								<div class = "middle-dark-tooltip"></div>
								<div class = "middle-light-tooltip"></div>
							</span>
						</div>
						<a class="button" href = "https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab143/6158036?redirectedFrom=fulltext" target = "_blank">Paper</a>
						<a class = "button" href = "https://CRAN.R-project.org/package=hierBipartite" target = "_blank">Software</a>
						<a class = "button" href = "https://github.com/CalvinTChi/hierBipartite" target = "_blank">Code</a>
					</div>
				</div>
			</div>
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/research1.jpg" height="225px">
				</div>
				<div class = "work-text">
					<h2>Admixture mapping reveals evidence of differential multiple sclerosis risk by genetic ancestry</h2>
					<p><b>Calvin Chi</b>, Xiaorong Shao, Brooke Rhead, Evangelina Gonzalez, Jessica B. Smith, Anny H. Xiang, Jennifer Graves, Amy Waldman, Timothy Lotze, Teri Schreiner, Bianca Weinstock-Guttman, Gregory Aaen, Jan-Mendelt Tillema, Jayne Ness, <i>et al.</i> </p>
					<p>PLoS Genetics 15(1) 2019</p>
					<br>
					<br>
					<div class = "button-panel">
						<div type = "button" class = "button-abstract">Abstract
							<span class = "abstract-bubble">
								Multiple sclerosis (MS) is an autoimmune disease with high prevalence among populations of northern European ancestry. Past studies have shown that exposure to ultraviolet radiation could explain the difference in MS prevalence across the globe. In this study, we investigate whether the difference in MS prevalence could be explained by European genetic risk factors. We characterized the ancestry of MS-associated alleles using RFMix, a conditional random field parameterized by random forests, to estimate their local ancestry in the largest assembled admixed population to date, with 3,692 African Americans, 4,915 Asian Americans, and 3,777 Hispanics. The majority of MS-associated human leukocyte antigen (HLA) alleles, including the prominent HLA-DRB1*15:01 risk allele, exhibited cosmopolitan ancestry. Ancestry-specific MS-associated HLA alleles were also identified. Analysis of the HLA-DRB1*15:01 risk allele in African Americans revealed that alleles on the European haplotype conferred three times the disease risk compared to those on the African haplotype. Furthermore, we found evidence that the European and African HLA-DRB1*15:01 alleles exhibit single nucleotide polymorphism (SNP) differences in regions encoding the HLA-DRB1 antigen-binding heterodimer. Additional evidence for increased risk of MS conferred by the European haplotype were found for HLA-B*07:02 and HLA-A*03:01 in African Americans. Most of the 200 non-HLA MS SNPs previously established in European populations were not significantly associated with MS in admixed populations, nor were they ancestrally more European in cases compared to controls. Lastly, a genome-wide search of association between European ancestry and MS revealed a region of interest close to the ZNF596 gene on chromosome 8 in Hispanics; cases had a significantly higher proportion of European ancestry compared to controls. In conclusion, our study established that the genetic ancestry of MS-associated alleles is complex and implicated that difference in MS prevalence could be explained by the ancestry of MS-associated alleles.
								<div class = "middle-dark-tooltip"></div>
								<div class = "middle-light-tooltip"></div>
							</span>
						</div>
						<a class = "button" href = "http://calvintchi.github.io/pdf/ASHG.2017.Calvin.pdf" target = "_blank">Poster</a>
						<a class="button" href = "https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007808" target = "_blank">Paper</a>
					</div>
				</div>
			</div>
		</div>
	</section>
	<section id = "projects-section" class = "gray-section">
		<div class = "section-title">Projects</div>
		<div class = "work">
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/project5.jpg" height="210px" vertical-align=middle>
				</div>
				<div class = "work-text">
					<h2>HLA Allele Imputation with Multitask Deep Convolutional Neural Network</h2>
					<div class = "tag-panel">
					<br>
						<div class = "tag">Research</div>
						<div class = "tag">Computational Biology</div>
						<div class = "tag">2020</div>
					</div>
					<br>
	  					<li>Developed multitask convolutional neural network for HLA imputation from phased genotype data.</li>
	  					<li>On the T1DGC test dataset, achieved 97.6\% imputation accuracy, which is comparable to state-of-the-art performance.</li>
	  				<br>
					<div class = "button-panel">
						<a href="https://www.biorxiv.org/content/10.1101/2021.06.03.447012v1" class = "button" target="_blank">Paper</a>
						<a href="http://calvintchi.github.io/pdf/ASHG_2019_Calvin.pdf" class = "button" target="_blank">Poster</a>
						<a href="https://github.com/CalvinTChi/HLA_imputation" class = "button" target="_blank">Code</a>
					</div>
				</div>
			</div>
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/project4.jpg" height="210px" vertical-align=middle>
				</div>
				<div class = "work-text">
					<h2>Embedding-Augmented Deep CNN for PudMed Journal Recommendation</h2>
					<div class = "tag-panel">
					<br>
						<div class = "tag">Class Project</div>
						<div class = "tag">NLP</div>
						<div class = "tag">2018</div>
					</div>
					<br>
	  					<li>Journal detection from PubMed abstract with 415,381 programmatically-collected abstracts</li>
	  					<li>Compared multitask and embedding-augmented CNNs with output space of 1,548 journals</li>
	  					<li>Best performance when CNN input augmented with topic and impact factor embeddings, with accuracy 23.7% and 90% of true journals in top 60 recommendations</li>
	  				<br>
					<div class = "button-panel">
						<a href="http://calvintchi.github.io/pdf/INFO259_final_report.pdf" class = "button" target="_blank">Report</a>
						<a href="http://calvintchi.github.io/pdf/INFO259_poster.pdf" class = "button" target="_blank">Poster</a>
						<a href="https://github.com/CalvinTChi/NLP259" class = "button" target="_blank">Code</a>
					</div>
				</div>
			</div>
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/project3.jpg" height="210px" vertical-align=middle>
				</div>
				<div class = "work-text">
					<h2>Data Augmentation using GAN for Breast Cancer Classification</h2>
					<div class = "tag-panel">
					<br>
						<div class = "tag">Class Project</div>
						<div class = "tag">Computer Vision</div>
						<div class = "tag">2018</div>
					</div>
					<br>
	  					<li>Investigated whether augmenting data with GANs could improve histology breast cancer classification with Resnet-18 re-trained on 5,547 breast histology images. Dataset provided by <a target="_blank" style="color:#3399ff; text-decoration: none;" href="https://www.kaggle.com/simjeg/lymphoma-subtype-classification-fl-vs-cll">Kaggle</a></li>
	  					<li>DCGAN most effective at generating realistic histology images when kernel size is divisible by stride length in the generator</li>
	  					<li>Augmentation with ~400 DCGAN images improved accuracy and precision by 5% and 12% respectively, but recall decreased by nearly 15%</li>
	  				<br>
					<div class = "button-panel">
						<a href="http://calvintchi.github.io/pdf/CS280_report.pdf" class = "button" target="_blank">Report</a>
						<a href="https://github.com/CalvinTChi/CS280_Project" class = "button" target="_blank">Code</a>
					</div>
				</div>
			</div>
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/project2.jpg" height="210px" vertical-align=middle>
				</div>
				<div class = "work-text">
					<h2>Analyzing the Effect of Salary on Employee Attrition</h2>
					<div class = "tag-panel">
					<br>
						<div class = "tag">Class Project</div>
						<div class = "tag">Causal Inference</div>
						<div class = "tag">2017</div>
					</div>
					<br>
	  					<li>Causal inference of effect of salary on employee attrition for simulated <a target="_blank" style="color:#3399ff; text-decoration: none;" href="https://www.kaggle.com/analystanand/employee-attrition">Kaggle</a> dataset</li>
	  					<li>Variable importance ranking of features for employee attrition</li>
	  					<li>Causal inference with DAGs, ensemble learning, and TMLE</li>
	  				<br>
					<div class = "button-panel">
						<a href="http://calvintchi.github.io/pdf/PH252D_report.pdf" class = "button" target="_blank">Report</a>
						<a href="http://calvintchi.github.io/pdf/Cantore_Chi_Davis_Lee_MP.pdf" class = "button" target="_blank">Presentation</a>
					</div>
				</div>
			</div>
			<div class = "projects-panel">
				<div class = "work-photo">
					<img src="assets/img/project1.jpg" height="210px" vertical-align=middle>
				</div>
				<div class = "work-text">
					<h2>Bearmaps</h2>
					<div class = "tag-panel">
					<br>
						<div class = "tag">Class Project</div>
						<div class = "tag">Data Structures</div>
						<div class = "tag">2016</div>
					</div>
					<br>
	  					<li>Mapping application for Berkeley, CA; implemented in Java</li>
	  					<li>Rastering with quad tree</li>
	  					<li>Routing via A* algorithm; location search autocompletion with trie</li>
	  				<br>
					<div class = "button-panel">
						<a href="http://calvin-bearmap.herokuapp.com/map.html" class = "button" target="_blank">App</a>
						<a href="http://datastructur.es/sp16/materials/proj/proj3/proj3.html" class = "button" target="_blank">Specs</a>
					</div>
				</div>
			</div>
		</div>
	</section>
	<section id = "resources-section" class = "white-section">
		<div class = "section-title">Resources</div>
		<div id="resources-intro">
			<p>During graduate school, I found it helpful to actively take notes on what I learned, in the same spirit as the Feynman technique. These notes include mathematical derivations and re-implementation of well-known algorithms, and are useful when I find existing resources inadequate in providing a good understanding. I continue to write notes as a life-long learner, and am posting these notes in case they are helpful. Feedback and corrections via email welcome.</p>
		</div>
		<div class = "resources-panel">
			<h2>Classical Machine Learning</h2>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/classical_machine_learning/2020/08/02/Principal-Component-Analysis.html">Principal component analysis</a></li>
		  		<li><a target="_blank" style="color:#3399ff" href="http://calvintchi.github.io/pdf/SVM.pdf">Support vector machine</a></li>
		  		<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/classical_machine_learning/2020/08/16/lwlr.html">Locally-weighted logistic regression</a></li>
		  		<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/classical_machine_learning/2020/09/24/collaborative_filtering">Collaborative filtering</a></li>
		  		<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/classical_machine_learning/2021/03/13/t-SNE">t-SNE</a></li>
		  	<br>
	  	</div>
	  	<div class = "resources-panel">
			<h2>Statistics</h2>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/statistics/2020/08/01/EM-Algorithm">Expectation-maximization algorithm</a></li>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/statistics/2020/08/20/multiple_hypothesis_testing">Multiple hypothesis testing</a></li>
		  	<br>
	  	</div>
	  	<div class = "resources-panel">
			<h2>Deep Learning</h2>
				<li><a target="_blank" style="color:#3399ff" href="http://calvintchi.github.io/deep_learning/2020/08/20/neural_networks">Neural networks</a></li>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/deep_learning/2020/08/09/AlexNet.html">AlexNet</a></li>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/deep_learning/2020/08/09/ResNet">ResNet</a></li>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/deep_learning/2020/09/23/generative_adversarial_network">Generative adversarial network</a></li>
		  	<br>
	  	</div>
	  	<div class = "resources-panel">
			<h2>Natural Language Processing</h2>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/nlp/2020/08/13/word2vec">word2vec</a></li>
		  	<br>
	  	</div>
	  	<div class = "resources-panel">
			<h2>Algorithms</h2>
				<li><a target="_blank" style="color:#3399ff" href="https://calvintchi.github.io/algorithms/2021/03/20/optimal_stopping">Optimal Stopping</a></li>
		  	<br>
	  	</div>
	</section>

	<section id = "end-section" class = "gray-section">
		<p>
			<a href="https://www.easycounter.com/">
				<img src="https://www.easycounter.com/counter.php?calvinchi" border="0" alt="Website Hit Counters">
			</a>
			<i>unique visitors since Mar. 2018</i>
			<br>
			<i>Last update: Jun. 2021</i>
		</p>
	</section>
</body>

</html>
